vortex_msgs/msg/LightInput.msg

```diff
@@ -1 +1,4 @@
-int32 light_intensity
+# Message for toggling lights on ROV
+# List of strings contain the name of the light(s)
+string[] toggle_light
+uint8[] intensity
```

Comment:
Update documentation here. Mention that the indices of the lists correspond (if they do). Specify the range for intensity values. Is it 0-255?